src/App.tsx

```diff
@@ -1,13 +1,49 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+// import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { ToodoList } from './components/TodoList';
+import { FilterOptions } from './utils/FilterOptions';
+import cn from 'classnames';
+
+function getFilteredTodos(todos: Todo[], filterOption: FilterOptions): Todo[] {
+  return todos.filter(todo => {
+    switch (filterOption) {
+      case FilterOptions.Active:
+        return !todo.completed;
+
+      case FilterOptions.Completed:
+        return todo.completed;
+
+      default:
+        return true;
+    }
+  });
+}
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todoos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
```

Comment:
It would be nice to create a separate enum for the list of possible errors and use it. Here and below in useEffect

Reply:
Good idea 👍 
Use `enum` for ur error.
```suggestion
  const [error, setError] = useState<ErrorType>(ErrorType.Default);
```